build/common.props

```diff
@@ -16,8 +16,8 @@
     <PackageReference Include="Internal.AspNetCore.Sdk" Version="$(InternalAspNetCoreSdkVersion)" PrivateAssets="All" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework' AND '$(OutputType)'=='library'">
-    <PackageReference Include="NETStandard.Library" Version="$(BundledNETStandardPackageVersion)" />
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'">
```

Comment:
Actually, you might need NS.Library.NETFramework to compile projects in `src` as I did in DataProtection. I'm fine with PrivateAssets=All for now, though I'm getting mixed messages about whether this package will be automatic or not. See https://github.com/dotnet/sdk/issues/1220#issuecomment-301956539 and https://github.com/dotnet/standard/issues/328#issuecomment-299579532